portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,79 +43,88 @@
   private static class Comment {
     private final String email, comment, date;
     private final long id;
-    public Comment(String email, String comment, String date, long id){
+
+    public Comment(String email, String comment, String date, long id) {
       this.email = email;
       this.comment = comment;
       this.date = date;
       this.id = id;
     }
   }
+
   private static class Response {
     private List<Comment> comments;
     private int lastPage;
+
     public Response(List<Comment> comments, int lastPage) {
       this.comments = comments;
       this.lastPage = lastPage;
     }
   }
+
   private static final String COMMENT_DATE_FORMAT = "MMM dd,yyyy HH:mm";
   private DatastoreService datastore;
   private PolicyFactory sanitizer;
   private WordCountUpdater wordCountUpdater;
-  
+
   @Override
   public void init() {
     datastore = DatastoreServiceFactory.getDatastoreService();
-    sanitizer = Sanitizers.FORMATTING.and(Sanitizers.BLOCKS).and(Sanitizers.STYLES).and(Sanitizers.LINKS);
+    sanitizer =
+        Sanitizers.FORMATTING.and(Sanitizers.BLOCKS).and(Sanitizers.STYLES).and(Sanitizers.LINKS);
     wordCountUpdater = new WordCountUpdater();
   }
-  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int commentsPerPage = Integer.parseInt(request.getParameter("comments_per_page"));
     int pgNumber = Integer.parseInt(request.getParameter("pg_number")) - 1;
     Query commentsQuery = new Query("Comment").addSort("time_millis", SortDirection.DESCENDING);
-    List<Entity> results = 
-      datastore.prepare(commentsQuery).asList(
-        FetchOptions.Builder
-          .withLimit(commentsPerPage)
-          .offset(commentsPerPage * pgNumber));
+    List<Entity> results =
+        datastore
+            .prepare(commentsQuery)
+            .asList(
+                FetchOptions.Builder.withLimit(commentsPerPage).offset(commentsPerPage * pgNumber));
```

Comment:
Looks like these are tabbed an extra space

Reply:
Darn, just realized everything is tabbed an extra space. Trying to configure the formatter somehow.